src/class.ziprecipes.php

```diff
@@ -298,8 +298,8 @@ public static function zrdn_format_recipe($recipe) {
 				'print_permalink_hide' => get_option('zlrecipe_printed_permalink_hide'),
 				'copyright' => get_option('zlrecipe_printed_copyright_statement')
 		);
-
-		return Util::view("recipe", $viewParams);
+        $custom_template = apply_filters('zrdn__custom_templates_show_tpl', false, $viewParams);
```

Comment:
Again, not a fan of acronyms like `tpl` because it can get quite out of hand.
This filter should be called `zrdn__custom_templates_get_formatted_recipe`.